create_key/create_key.go

```diff
@@ -102,7 +102,7 @@ func GenerateKeyPair(basename, password string) error {
 	}
 	defer func() {
 		if err := pubFile.Close(); err != nil {
-			log.Errorf("Error closing file: %s\n", err)
+			fmt.Printf("Error closing file: %s\n", err)
```

Comment:
Errors should go to the standard error stream.

```suggestion
			fmt.Fprintf(os.Stderr, "Error closing file: %v\n", err)
```

Note change to `%v` too. It may be worth while investigating whether one should use `%w` to format the error string.

Note: This also applies to all other changes from `log.Error()` to `fmt.Printf` in this PR.